ipaclient/ntpconf.py

```diff
@@ -159,9 +159,12 @@ def synconce_ntp(server_fqdn, debug=False):
         timeout = 15
         root_logger.info('Attempting to sync time using ntpd.  '
                          'Will timeout after %d seconds' % timeout)
-        ipautil.run(args, timeout=timeout)
+        args[0:0] = [paths.BIN_TIMEOUT, str(timeout)]
+        ipautil.run(args)
```

Comment:
I'd personally move the try-except only to include ipautil.run and perhaps the log and would create the args at one point right away (QuantifiedCode is working atm but I'd guess it'd be ranting about this).